sopel/cli/plugins.py

```diff
@@ -0,0 +1,397 @@
+    list_parser.add_argument(
+        '-n', '--name-only',
+        help='Display only plugin names',
+        dest='name_only',
+        action='store_true',
+        default=False)
+
+    # sopel-plugin disable
+    disable_parser = subparsers.add_parser(
+        'disable',
+        help="Disable a Sopel plugins",
+        description="""
+            Disable a Sopel plugin by its name, no matter where it comes from.
+            It is not possible to disable the ``coretasks`` plugin.
+        """)
+    utils.add_common_arguments(disable_parser)
+    disable_parser.add_argument('name', help='Name of the plugin to disable')
+    disable_parser.add_argument(
+        '-f', '--force', action='store_true', default=False,
+        help="""
+            Force exclusion of the plugin. When ``core.enable`` is defined, a
+            plugin may be disabled without being excluded. In this case, use
+            this option to force its exclusion.
+        """)
+    disable_parser.add_argument(
+        '-r', '--remove', action='store_true', default=False,
+        help="""
+            Remove from ``core.enable`` list if applicable.
+        """)
+
+    # sopel-plugin enable
+    enable_parser = subparsers.add_parser(
+        'enable',
+        help="Enable a Sopel plugins",
+        description="""
+            Enable a Sopel plugin by its name, no matter where it comes from.
+            The ``coretasks`` plugin is always enabled. By default, a plugin
+            that is not excluded is enabled, unless at least one plugin is
+            defined in the ``core.enable`` list. In that case, Sopel uses
+            a "allow-only" policy for plugins, and enabled plugins must be
+            added to this list.
+        """)
+    utils.add_common_arguments(enable_parser)
+    enable_parser.add_argument('name', help='Name of the plugin to enable')
+    enable_parser.add_argument(
+        '-a', '--allow-only',
+        dest='allow_only',
+        action='store_true',
+        default=False,
+        help="""
+            Enforce allow-only policy, adding the plugin to the ``core.enable``
+            list.
+        """)
+
+    return parser
+
+
+def handle_list(options):
+    """List Sopel plugins"""
+    settings = utils.load_settings(options)
+    no_color = options.no_color
+    name_only = options.name_only
+    enabled_only = options.enabled_only
+    disabled_only = options.disabled_only
+
+    for name, info in plugins.get_usable_plugins(settings).items():
+        plugin, is_enabled = info
+
+        if enabled_only and not is_enabled:
+            # hide disabled plugins when displaying enabled only
+            continue
+        elif disabled_only and is_enabled:
+            # hide enabled plugins when displaying disabled only
+            continue
+
+        description = {
+            'name': name,
+            'status': 'enabled' if is_enabled else 'disabled',
+        }
+
+        # option meta description from the plugin itself
```

Comment:
This is a guess; I'm not 100% sure this is what you meant, but I try anyway!

```suggestion
        # optional meta description from the plugin itself
```

Reply:
Oh, yeah, that was my intent.